### ISSUE TITLE
1.7 install via cargo fails

### ISSUE THREAD

**Issue by toinbis** [cloudflare/wrangler#1038]
#### Environment

* operating system:
* output of `rustc -V`: rustc 1.37.0 (eae3437df 2019-08-13)
* output of `node -v`: v10.16.2
* output of `wrangler -V`: 👷 ✨  wrangler 1.7.0 (installed via npm, was 1.6.0 prior0)

#### Steps to reproduce
1. $cargo install wrangler

#### What did you expect to see?
Wrangler to be installed.

#### What did you see instead?
```Compiling httparse v1.3.4
   Compiling itoa v0.4.5
error: `core::slice::<impl [T]>::len` is not yet stable as a const fn
   --> /Users/t0073/.cargo/registry/src/github.com-1ecc6299db9ec823/bytes-0.5.4/src/bytes.rs:130:18
    |
130 |             len: bytes.len(),
    |                  ^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `bytes`.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `wrangler v1.7.0`, intermediate artifacts can be found at `/var/folders/dd/b3p0n8d51252zbh3h25rhrt00000gn/T/cargo-installmS6759`

Caused by:
  build failed

**Comment by EverlastingBugstopper** [comment]
Hey @toinbis, sorry about that. Try running `rustup update` and see if that fixes it - Rust 1.41 is out now and i think that should help.

**Comment by toinbis** [comment]
Thanks, @EverlastingBugstopper  - it helped ineed! :)
